ArduCopter/autoyaw.cpp

```diff
@@ -208,7 +209,7 @@ void Mode::AutoYaw::set_roi(const Location &roi_location)
 void Mode::AutoYaw::set_rate(float turn_rate_cds)
```

Comment:
To be consistent with the other AutoYaw functions I think it is worth adding units to this function name e.g. `set_rate_cdegs()`